tools/ci_status_report.py

```diff
@@ -85,8 +85,10 @@ def filter_jobs(response: requests.Response) -> typing.Iterator[Job]:
         for ref in all_refs:
             organization = ref["org"]
             repository = ref["repo"]
-            base_ref = ref["base_ref"]
             if (organization, repository) in TRACKED_REPOSITORIES:
+                base_ref = ref.get("base_ref")
+                if not base_ref:
```

Comment:
wouldn't it still raise an exception?
I mean, in case it's not available in the dictionary ``base_ref`` will be ``None`` which still gets into raising the exception

Reply:
either way the ``ci-status-report-dry-run`` job will answer that question

Reply:
got it. it will only apply to our repos
that's good